remove-git-ref/action.yml

```
@@ -8,29 +8,28 @@ description: >
   Note: removing tags is not supported yet.
 
 inputs:
-  owner:
-    description: Repository owner
-    required: true
-  repo:
-    description: Repository name
+  repository:
+    description: Repository name (<owner>/<repo>)
     required: true
   ref:
     description: Git reference to remove
+    required: true
   token:
     description: GitHub personal access token (PAT)
     required: true
 
 runs:
   using: composite
   steps:
-    - name: Remove reference ${{ inputs.ref }} from ${{ inputs.owner }}/${{ inputs.repo }}
+    - name: Remove reference ${{ inputs.ref }} from ${{ inputs.repository }}
       uses: actions/github-script@v6
       continue-on-error: true
       with:
         github-token: ${{ inputs.token }}
         script: |
+          const fullRepoName = "${{ inputs.repository }}".split("/")
           await github.request('DELETE /repos/{owner}/{repo}/git/refs/heads/{ref}', {
```

Just use the variable as it is 
```suggestion
          await github.request('DELETE /repos/{repository}/git/refs/heads/{ref}', {
```

---

Fixed.